src/main/java/org/broadinstitute/consent/http/service/ElectionService.java

```diff
@@ -510,8 +508,8 @@ private void setGeneralFields(Election election, String referenceId, ElectionTyp
                 break;
             case DATA_ACCESS:
             case RP:
-                Document dar = dataAccessRequestService.getDataAccessRequestByReferenceIdAsDocument(referenceId);
-                List<Integer> datasetIdList = DarUtil.getIntegerList(dar, DarConstants.DATASET_ID);
+                DataAccessRequest dar = dataAccessRequestService.findByReferenceId(referenceId);
+                List<Integer> datasetIdList = Objects.nonNull(dar) && Objects.nonNull(dar.getData()) ? dar.getData().getDatasetIds() : Collections.emptyList();
```

Comment:
Just a thought - we do this null check quite often. Not for this PR, but a nice refactoring would be to have a `getDatasetIds()` method on DataAccessRequest that would do all the checking for us. That in itself is a pretty big change across lots of classes.